dask/dataframe/io/parquet.py

```diff
@@ -343,6 +343,11 @@ def to_parquet(path, df, compression=None, write_index=None, has_nulls=True,
         Construct directory-based partitioning by splitting on these fields'
         values. Each dask partition will result in one or more datafiles,
         there will be no global groupby.
+    times: 'int64' (default), or 'int96':
+        In "int64" mode, datetimes are written as 8-byte integers, us
+        resolution; in "int96" mode, they are written as 12-byte blocks, with
+        the first 8 bytes as ns within the day, the next 4 bytes the julian day.
+        'int96' mode is included only for compatibility.
```

Comment:
I recommend expanding `us` to microseconds and `ns` to nanoseconds.  The last sentence makes it sound like this second form is inferior and should be avoided.  Is this the intent?

Reply:
Yes, in the sense that it is not the default, and has worse performance in both reading and writing. If you want spark to read it, though, you have no choice, because spark does not fulfil the parquet spec...

Reply:
INT96 Timestamps were also never clearly specified and are now considered deprecated, Spark 2.3+ will also support the INT64 timestamps; support for them was recently merged.